vwifi.c

```diff
@@ -1782,6 +1873,7 @@ static void owl_free(void)
     list_for_each_entry_safe (vif, safe, &owl->vif_list, list)
         owl_delete_interface(vif);
 
+    kfree(owl->blocklist);
```

Comment:
@jserv 
I do the release of blocklist's kernel memory within function `owl_free`.
I think blocklist is now treated as a global content so I put it here.